gui/resources/sitecustomize.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import absolute_import
```

Comment:
Please do not use this future import without reason.  It makes for very confused reading on 2.x.  Since that's in a few files I only mention it here.